packages/bpk-component-navigation-bar/src/BpkNavigationBar.module.scss

```diff
@@ -43,6 +43,10 @@
     tokens.$bpk-surface-default-day
   );
 
+  &--justified {
+    justify-content: space-between;
+  }
```

Comment:
![Screenshot 2024-05-03 at 14 10 40](https://github.com/Skyscanner/backpack/assets/89925955/cbb533ac-af3d-4685-b86c-464e8bc12f7e)

Isn't this already set?

Reply:
Yes! good spot removed 😅